### PR TITLE
refactor(logging): unify and tidy up the logging code [MS-984]

### PR DESCRIPTION
**Pull request by GrzegorzDerdak**
I want to merge this change because it clears up the code a little bit.

# Pull Request Checklist

- [x] Test the changes locally to ensure they work as expected.
- [ ] Document the testing process and results in the pull request description. (Screen recording, screenshot etc)
- [ ] Include new tests for any new functionality or significant changes.
- [ ] Ensure that tests cover edge cases and potential failure points.
- [ ] Document the impact of the changes on the system, including potential risks and benefits.
- [ ] Provide a rollback plan in case the changes introduce critical issues.
- [x] Update documentation to reflect any changes in functionality.

Related task: [MS-948](https://mirumee.atlassian.net/browse/MS-948)

[MS-948]: https://mirumee.atlassian.net/browse/MS-948?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/infrastructure/src/logging/service.ts [hunk 1]
```diff
@@ -1,68 +1,45 @@
...
+ * @see {@link Logger} for the logger type.
+ * @see {@link LogLevel} for available log levels.
+ * @see {@link LoggerOptions} for available logger options.
+ *
+ * @example
+ * import { logger } from "@nimara/infrastructure/logging/service";
+ * logger.info("Hello world");
+ * logger.error("Something went wrong");
+ */
+export const logger: Logger = getLogger();
```
Comment (piotrgrundas) [inline packages/infrastructure/src/logging/service.ts:45]: I think each serving using it should `getLogger` itself and there we would define the `name`. 
Look at stripe app.

**Comment by piotrgrundas** [comment]
Looks good. 
THO, at some point we should switch to (winston)[https://github.com/winstonjs/winston].

**Comment by GrzegorzDerdak** [comment]
Sure, we can always add another provider by adding a winston.ts and justing the code accordingly.

**Review by gajda-w — APPROVED** [review]
